src/playbacks/no_op/no_op.js

```diff
@@ -25,13 +25,16 @@ export default class NoOp extends Playback {
     this.$el.html(this.template({message: playbackNotSupported}))
     this.$el.append(style)
     this.trigger(Events.PLAYBACK_READY, this.name)
-    this._options.autoPlay && this.play()
+    const showForNoOp = !!(this.options.poster && this.options.poster.showForNoOp);
+    if (this.options.autoPlay || !showForNoOp) {
+      this.play()
+    }
```

Comment:
One of the reasons for this PR is that to stop NoOp animations when it's not shown (e.g. autoPlay is false). Currently it eats up 7-10% CPU on i5 Haswell processor and it's only 1 instance of the player. If user would go on a page with multiple intances of Clappr and will get NoOp playback it will eat up like 30% CPU which is definitely not good. What can we do about it?